pkgs/development/python-modules/timetagger/default.nix

```diff
@@ -9,13 +9,13 @@
 
 python3Packages.buildPythonPackage rec {
   pname = "timetagger";
-  version = "22.3.1";
+  version = "22.4.2";
 
   src = fetchFromGitHub {
     owner = "almarklein";
     repo = pname;
-    rev = "v${version}";
-    sha256 = "sha256-pHogDjqXuoQp5afSnPvMPaKoBtPE6u3kMi87SzY5yoU=";
+    rev = "refs/tags/v${version}";
```

Comment:
I guess taht is not necessary, is it?

Reply:
The update script has been updated in https://github.com/NixOS/nixpkgs/commit/7f5915b04801f383a3504a011601e2fb09dfedf8 to always use ref/tags/ for specifying rev.

Reply:
Ah, cool.